js/clinicalTimeline.js

```diff
@@ -218,7 +220,7 @@ window.clinicalTimeline = (function(){
       if (getZoomLevel(minDays, maxDays, width * zoomFactor) !== "days") {
         // add brush overlay
         var xScale = d3.time.scale()
-           .domain([beginning, ending])
+           .domain([minDays, maxDays])
```

Comment:
Ah yes, this is a mistake on my part. This wasn't supposed to be committed. I was just trying something and forgot to undo.